command/agent/auth/auth.go

```diff
@@ -281,6 +280,7 @@ func (ah *AuthHandler) Run(ctx context.Context, am AuthMethod) error {
 			}
 
 			am.CredSuccess()
+			backoff.reset()
 		}
```

Comment:
Could we have just reset the backoff here with `backoff = initialBackoff`?

Reply:
My main motivation for the function was that the action of resetting is sprinkled about in various places so I wanted to ensure consistent handling.